release-content/0.14/release-notes/10690_Upgrade_to_Taffy_04.md

```diff
@@ -1 +1,5 @@
-TODO
+[`taffy`](https://crates.io/crates/taffy), the UI layout crate that we rely on, has had a major update.
+In addition to various bug fixes, this brings two new pieces of functionality to users of `bevy_ui`:
+
+1. `Display::Block`: use the [CSS block layout model] to layout UI nodes. Elements will start on a new line, and take up the whole width. Great for in-line images!
```

Comment:
Are you missing a link here?

Reply:
I-

This is specific to line 4. _Really_ don't know why it included lines 1 through 3.

Reply:
If you select multiple lines on the left you can leave comments on the whole block :) Easy to do by accident if you're not aware of the feature.